src/racing/math_util.c

```diff
@@ -426,18 +464,20 @@ void func_802B5D64(uintptr_t arg0, s16 arg1, s16 arg2, s32 arg3) {
     }
 }
 
-void func_802B5F00(Mat4 arg0, f32 arg1) {
-    arg0[0][0] *= arg1;
-    arg0[1][0] *= arg1;
-    arg0[2][0] *= arg1;
-    arg0[0][1] *= arg1;
-    arg0[1][1] *= arg1;
-    arg0[2][1] *= arg1;
-    arg0[0][2] *= arg1;
-    arg0[1][2] *= arg1;
-    arg0[2][2] *= arg1;
+// multiply a matrix with a number
+void mtxf_multiplication_number(Mat4 mat, f32 coef) {
```

Comment:
`mtxf_scale_3x3` ?

Reply:
but he use a mat4 and in general we don't touch the column four and the line four when you need to multiply with a position (vec3)

Reply:
What about scale rather than multiply? Or do you think multiply is more accurate?

Reply:
by some research it's appear to be scale